packages/components/table/src/table.tsx

```diff
@@ -30,6 +36,10 @@ const Table = forwardRef<"table", TableProps>((props, ref) => {
     ref,
   });
 
+  const {isVirtualized, rowHeight = 40, maxTableHeight = 600} = props;
+
+  const shouldVirtualize = values.collection.size > 50 || isVirtualized;
+
```

Comment:
@vinroger let's add a prop to make this component virtualized (at least for v.2.7.0) then we can apply this automatically, but only when we are sure it is stable, do the same with the `select`

Reply:
currently `isVirtualized` is the prop to make it virtualized. in select, there is also such logic (i.e. virtualized automatically when size > 50)